src/app.js

```diff
@@ -1 +1,32 @@
 'use strict';
+
+const readline = require('readline');
+
+const { generateNumber } = require('./generateNumber');
+const { checkUsersTry } = require('./checkUsersTry');
+
+const launchGame = () => {
+  const terminal = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
```

Comment:
As per requirements, you should have a separate module for input/output. Create it and move your `readline` stuff there.